spatial-tracking-explainer.md

```diff
@@ -177,6 +177,45 @@ function onSessionStarted(session) {
 }
 ```
 
+### Application-supplied transforms
+Frequently developers will want to provide an additional, artificial transform on top of the user's tracked motion to allow the user to navigate larger scenes than their tracking systems or physical space allows. This effect is traditionally accomplished by mathematically combining the API-provided transform with the desired additional application transforms. It is useful to allow the API do handle it, however, because it ensures that all tracked values are transformed consistently, allowing the user's viewpoint and inputs to stay in sync.
```

Comment:
Nit: this is a bit confusing sounding to me...

>It is useful to allow the API do handle it, however, however, because it ensures that all tracked values are transformed consistently, allowing the user's viewpoint and inputs to stay in sync.

How about this instead:

Nit: this is a bit confusing sounding to me...

> WebXR offers developers a simplification to ensure that all tracked values are transformed consistently, allowing the user's viewpoint and inputs to stay in sync.